guides/mapping-and-blending/avoiding-css-conflicts/README.md

```diff
@@ -113,11 +94,15 @@ We recommend the following rules when using BEM selectors in Starcounter apps.
 }
 ```
 
+* __Never nest blocks inside blocks and elements inside elements.__
+
+ If there’s need for more nesting, it means there’s too much complexity and the elements should be stripped down into smaller blocks.
+
```

Comment:
I've borrowed this principle from https://m.alphasights.com/how-we-use-bem-to-modularise-our-css-82a0c39463b0. Please check there for long explanation.

In this case, I guess that the advised code would be:

```css
<button class="app-menu-button">
    <span class="app-menu-button__description">Description</span>
    <i class="app-menu-button__icon fa fa-circle"></i>
</button>
```